### PR TITLE
initial changes of Razorpay Payment Button for Drupal

### PR DESCRIPTION
**Pull request by neera11**

### CODE REVIEW

File: payment/plugin.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+					}
+				};
+			});
+		}
+	});
+})(jQuery);
+
+
+
+function getPaymentButtons(){
```
Comment (sofiya02) [inline payment/plugin.js]: Kindly check the consistency for function nomenclature camelcase or snake case

File: payment/plugin.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	
+	var buttondata = CKEDITOR.ajax.load(Drupal.url('/button_json'));
```
Comment (sofiya02) [inline payment/plugin.js]: remove '/' as it breaking the url

File: README.md [hunk 3]
```diff
@@ -1 +1,164 @@
-# payment-button-drupal-plugin
+Payment Button Drupal Plugin
+==============
+
```
Comment (sofiya02) [inline README.md:3]: Do add following section
- support 
- License
- Change log

File: payment/plugin.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
+/*
+* Razorpay Payment Button Plugin
+* @author Razorpay <neeraj.s@razorpay.com>
+* @version 1.0
+*/
+(function ($) {
+	
+	CKEDITOR.plugins.add('payment', {
```
Comment (sofiya02) [inline payment/plugin.js:10]: Comment before  function is initiated

File: README.md (hunk 3 above)
Comment (neera11) [reply README.md:3]: Support and Change log are added in readme file
for License we have other file with name of LICENSE.txt

File: payment/plugin.js (hunk 2 above)
Comment (neera11) [reply payment/plugin.js]: removed

File: payment/plugin.js (hunk 1 above)
Comment (neera11) [reply payment/plugin.js]: changed function in snake case

File: payment/plugin.js (hunk 4 above)
Comment (neera11) [reply payment/plugin.js:10]: added

**Review by sofiya02 — APPROVED** [review]

**Review by sofiya02 — APPROVED** [review]

**Pull request merged**